athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/serde/v2/ListTablesRequestSerDe.java

```diff
@@ -59,6 +63,10 @@ protected void doRequestSerialize(FederationRequest federationRequest, JsonGener
             ListTablesRequest listTablesRequest = (ListTablesRequest) federationRequest;
 
             jgen.writeStringField(SCHEMA_NAME_FIELD, listTablesRequest.getSchemaName());
+
```

Comment:
why empty lines between?

Reply:
I didn't change that - it was like that before. But I will change.

Reply:
I wouldn't say that the extra separation is absolutely needed, but to me it makes it look cleaner to be able to see each field being written.

Reply:
this is not consistent with our styling, can you remove the extra lines?

Reply:
Removed extra blank lines.